TFT/src/User/API/UI/ListItem.c

```diff
@@ -33,18 +35,20 @@ char * getDynamicTextValue(uint8_t i)
 // save dynamic value (upto 7 digits) ( i : index of the value position, value:float value)
 void setDynamicValue(uint8_t i, float value)
 {
-  if (value < 1.0f)
-    sprintf(dynamic_text_value[i], "%.3f", value);
+  if (value < 10.0f)
```

Comment:
maybe it should be considered abs(value) and the number of decimal digits should be reduced by 1 in case the value is negative (there is a - sign)